scene/realistic-lighting-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/ComboBoxStringConverter.java

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Just a tidy up suggestion.

For both this class and the `LightingModeListCell` class: since these have a small amount of content, these can live at the bottom of the `RealisticLightingAndShadowsController` class, as we do in some of our other samples (see e.g. https://github.com/Esri/arcgis-runtime-samples-java/blob/e75be37b2eb83a672f57afc1b1cb64e5549506b3/map/create-and-save-map/src/main/java/com/esri/samples/create_and_save_map/CreateAndSaveMapController.java#L176). 

This ensures our samples remain clean (as you've done a great job doing separating out these converters), but also that there isn't a large amount of classes sitting within the sample package. It also shows our MVC pattern more clearly with the converter classes hidden within the controller class.

Reply:
@Rachael-E if this is the way forward, then there should be an issue to change up the samples where these are separated to their own files. Since I've started, all the samples using ListCell have had these in their individual files. 
I personally think that's the correct way to do it, since essencially ListCell is UI code unrelated to the controller, so putting it in it's own class reduces clutter in the controller and puts the Runtime code in the focus of what's going on there. After all, if we could do this easily, we'd probably do this in the FXML file as well.
Either way, there should at least be a clear guidance in the contribution guidelines, I think

Reply:
thanks @JonLavi, you're right, we currently don't have a consistent way this is handled in the samples and it would be good to reach a design pattern agreement. Either approach works: it's just a matter of deciding what is the best practice pattern: we should get feedback on that during a Java catch up meeting. I'm of the opinion in the instance where it's a standard converter it is fine to "hide" at the bottom of the controller class (in and basic cases won't even need to be in Class format at all). This then focusses the user on our Controller class and our API. If however there was a more "involved" conversion (such as that of GroupLayer sample) then it merits its own abstracted class.